src/components/coin.ts

```diff
@@ -165,15 +165,17 @@ export const createCoinLedgerEntry = async (
   Otherwise, update last_granted to CURRENT_TIMESTAMP.
 */
 export const updateUserBonusTableByUserId = async (userId: string, bonusType: BonusType): Promise<void> => {
-  const db = await openDB();
-  await db.run(
-    `
-    INSERT INTO user_coin_bonus (user_id, bonus_type, last_granted) VALUES (?, ?, CURRENT_TIMESTAMP)
-    ON CONFLICT(user_id, bonus_type)
-    DO UPDATE SET last_granted = CURRENT_TIMESTAMP`,
-    userId,
-    bonusType
-  );
+  try {
+    const db = await openDB();
+    await db.run(
+      `
+      INSERT INTO user_coin_bonus (user_id, bonus_type, last_granted) VALUES (?, ?, CURRENT_TIMESTAMP)
+      ON CONFLICT(user_id, bonus_type)
+      DO UPDATE SET last_granted = CURRENT_TIMESTAMP`,
+      userId,
+      bonusType
+    );
+  } catch (e) {}
```

Comment:
not sure why try catch is needed
if needed, should log smth

Reply:
Removed, this was from debugging that I forgot to remove, oops